org/mozilla/jss/provider/java/security/JSSKeyStoreSpi.java

```diff
@@ -215,6 +215,10 @@ public boolean engineContainsAlias(String alias) {
         return getAliases().contains(alias);
     }
 
+    /**
+     * @deprecated getUniqueID() in PrivateKey has been deprecated
```

Comment:
Per conversation with @edewata, I think we should un-deprecate these as well. So, please remove this annotation.